Software/GuitarPedal/Effect-Modules/nam_module.cpp

```diff
@@ -203,6 +206,12 @@ void NamModule::ProcessMono(float in) {
     if (GetParameterAsBool(6)) {
         ampOut =
             rtneural_wavenet.forward(input_arr[0]) * 0.4; // TODO Try this again, was sending the whole array, wants just the float
+
+        if (m_currentModelindex == 12) {
```

Comment:
TODO: redo my capture with more attention to my levels

Reply:
opted to just add a manual levelAdjust parameter per model that can be tweaked, it is probably going to be more useful and is certainly a bit easier